### PR TITLE
New Recycle Cheat Sheet

### PR DESCRIPTION
**Pull request by leemartie**
## Description of new Instant Answer, or changes

This cheat sheet is intended to aid the user in understanding what plastics can be recycled to help them sort their trash and make them more aware of the recyclability of the plastics they buy. The user is presented with definitions of plastic recycling numbers, how recyclable the plastic is by number, and examples of plastics by number.

https://duck.co/ia/view/recycle_cheat_sheet

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/recycle_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/recycle.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    "name": "Recycle",
+    "description": "Plastic resin number definitions, recyclability, and examples.",
+
+    "metadata": {
+        "sourceName": "Public Broadcasting Service (PBS)",
+        "sourceUrl" : "http://www-tc.pbs.org/strangedays/pdf/StrangeDaysSmartPlasticsGuide.pdf"
+    },
+
+    "aliases": [
+        "resin","resin numbers","resin number","plastic", "recycle numbers", "recycle number", "plastic number", "plastic numbers","recycling"
```
Comment (pjhampton) [inline share/goodie/cheat_sheets/json/recycle.json]: Can you please use a consistent spacing --> `"resin number", "plastic"` with one space after each comma

File: share/goodie/cheat_sheets/json/recycle.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                "key":"7",
+                "val":"lids, medical storage containers, electronics, most plastic baby bottles, 5-gallon water bottles, “sport” water bottles, metal food can liners, clear plastic “sippy” cups, and some clear plastic cutlery"
+                
+            }
+            
+            
+            
+            
+            
+
```
Comment (pjhampton) [inline share/goodie/cheat_sheets/json/recycle.json]: let's cut down on the whitespace

File: share/goodie/cheat_sheets/json/recycle.json (hunk 1 above)
Comment (leemartie) [inline share/goodie/cheat_sheets/json/recycle.json]: I used the DuckDuckGo JSON Beautifier & Validator (https://duckduckgo.com/?q=json+validator&ia=answer&atb=v60-3_e) to reformat the JSON file.  Does that work? Thanks.

File: share/goodie/cheat_sheets/json/recycle.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      },
+      {
+        "key": "7",
+        "val": "MIXED (OTHER)"
+      }
+    ],
+    "Recyclability": [
+      {
+        "key": "1",
+        "val": "PET is one of the most easily recycled plastics."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/recycle.json]: Lets remove periods at end of val fields as well as description

File: share/goodie/cheat_sheets/json/recycle.json [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+  "aliases": [
+    "resin",
+    "resin numbers",
+    "resin number",
+    "plastic",
+    "recycle numbers",
+    "recycle number",
+    "plastic number",
+    "plastic numbers",
+    "recycling"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/recycle.json]: Is recycling an appropriate alias for this cheatsheet?  I think it is too generic

Comment (leemartie) [inline share/goodie/cheat_sheets/json/recycle.json]: You might be right.  How does "plastic recycling" sound?

File: share/goodie/cheat_sheets/json/recycle.json (hunk 3 above)
Comment (leemartie) [inline share/goodie/cheat_sheets/json/recycle.json]: Not sure I follow. Why remove a period marking the end of a sentence?

Comment (jophab) [inline share/goodie/cheat_sheets/json/recycle.json]: Usually in cheatsheets single line val fields dont have peroid at the end. You can check other cheatsheets.

Comment (leemartie) [inline share/goodie/cheat_sheets/json/recycle.json]: Okay, thanks for explaining this to me.

File: share/goodie/cheat_sheets/json/recycle.json [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  "section_order": [
+    "Resin Numbers",
+    "Recyclability",
+    "Examples"
+  ],
+  "sections": {
+    "Resin Numbers": [
+      {
+        "key": "1",
+        "val": "POLYETHYLENE TEREPHTHALATE (PET)"
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/recycle.json]: @leemartie can we change this from all capitals to CamelCase?
For example, `"POLYETHYLENE TEREPHTHALATE (PET)"` can become `"PolyEthylene Terephthalate"`.

Comment (leemartie) [inline share/goodie/cheat_sheets/json/recycle.json]: Hi @sahildua2305. I have made the requested changes. Thanks.

File: share/goodie/cheat_sheets/json/recycle.json (hunk 4 above)
Comment (kirkins) [inline share/goodie/cheat_sheets/json/recycle.json]: I like recycling. If I search "recycling cheat sheet" this is probably what I'd be looking for.

Comment (kirkins) [inline share/goodie/cheat_sheets/json/recycle.json]: Ah, I didn't realize the change was already made. No sense in making another change. 👍

**Comment by kirkins** [comment]
Took a look through code and version on beta. LGTM 👍

**Pull request merged**